src/Integration.Vsix.UnitTests/SonarLintTagger/IssuesSnapshotTests.cs

```diff
@@ -158,6 +162,34 @@ public void ProjectName()
             GetValue(StandardTableKeyNames.ProjectName).Should().Be("MyProject");
         }
 
+        [TestMethod]
+        [DataRow(DaemonSeverity.Info, __VSERRORCATEGORY.EC_MESSAGE)]
+        [DataRow(DaemonSeverity.Minor, __VSERRORCATEGORY.EC_MESSAGE)]
+        [DataRow(DaemonSeverity.Major, __VSERRORCATEGORY.EC_WARNING)]
+        [DataRow(DaemonSeverity.Critical, __VSERRORCATEGORY.EC_WARNING)]
+        public void ToVsErrorCategory_NotBlocker_CorrectlyMapped(DaemonSeverity daemonSeverity, __VSERRORCATEGORY expectedVsErrorCategory)
```

Comment:
These tests are the same as those for the RuleSetGenerator, which the exception of the "invalid value" test.